### ISSUE TITLE
FileUploadOption in fileTransfer.d.ts

### ISSUE THREAD

**Issue by podger** [Microsoft/cordova-plugin-code-push#65]
Hi,
I've encountered an issue trying to compile my TypeScript code envolving the cordova file transfer plugin.
```
interface FileUploadOptions {
    ...

    /** A map of header name/header values. Use an array to specify more than one value. */
    headers?: Object[];
}
```

Shouldn't it be:
`    headers?: Object;
`
instead? This way is working fine. I looked up the [File Transfer Docs](https://cordova.apache.org/docs/it/3.1.0/cordova/file/filetransfer/filetransfer.html), (even if it's not updated) and doesn't mention array object in the headers param.

I guess the same is for `FileDownloadOptions` 

Am i missing something?

Thank you,
Bye

**Comment by lostintangent** [comment]
@podger If you change the d.ts to use an object instead of an object array, does your app compile then?

**Comment by podger** [comment]
@lostintangent yes it compiles perfectly using an `Object `type using tsc 1.0.3.0, gives error instead with `Object[]`

**Comment by lostintangent** [comment]
Resolved by #74
